dask/array/tests/test_array_core.py

```diff
@@ -67,6 +67,9 @@
 from dask.utils import SerializableLock, key_split, tmpdir, tmpfile
 from dask.utils_test import dec, hlg_layer_topological, inc
 
+if da._array_expr_enabled():
```

Comment:
These skips are necessary, we are using unsupported stuff in fixture definitions that get executed even if the file isn't marked properly